firecloud_project.py

```diff
@@ -353,71 +381,95 @@ def generate_config(context):
   project_name = context.properties.get('projectName', project_id)
   labels_obj = context.properties.get('labels', {})
 
+  # Save this template's version number and all parameters inputs to the project metadata to keep track of what
+  # operations were performed on a project.
+  labels_obj.update({
+    "firecloud-project-template-version" : "1"
```

Comment:
pull the version into a constant at the top of the file with comment to update for any major feature change

Reply:
👍